5001-6000/LIT5354Mashaf.xml

```diff
@@ -62,19 +68,67 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          </langUsage>
       </profileDesc>
       <revisionDesc>
-         <change who="DE" when="2019-03-15">Created entity</change>         
+         <change who="DE" when="2019-03-15">created entity</change>
+         <change who="DN" when="2025-05-19">added incipit, bibliography, witnesses, work structure</change> 
+        
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
-         <div type="bibliography"/>
-         <!---->
-         <listRelation>
-            <relation name="dcterms:creator"
-                      active="LIT5354Mashaf"
-                      passive="PRS12520TaklaH"/>
+         <div type="bibliography">
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Platonov2002Lies"/>                  
+               </bibl>
+            </listBibl>            
+            <listRelation>
+    
+            <relation name="dcterms:creator" active="LIT5354Mashaf" passive="PRS12520TaklaH"/>
+               <relation name="lawd:hasAttestation" active="LIT5354Mashaf" passive="IVEf27"/>
+               <relation name="lawd:hasAttestation" active="LIT5354Mashaf" passive="ANLcr79"/>
          </listRelation>
+         </div>
+         <div type="edition" xml:lang="gez">
+            <note>The following division into stanzas and text transacriptions are taken from
+               the edition <bibl><ptr target="bm:Platonov2002Lies"/></bibl>.</note>      
+            
+          <div type="edition">
+                  <div type="textpart" subtype="chapter" n="1" xml:id="ChapterOne">
+                     <label>Chapter 1</label>
+                     <ab>እኔ፡ እግዚአብሔር፡ በጽድቅ፡ ጸራኁህ፡ እጅኽንም፡ ያዝሁኽ፡ 
```

Comment:
Strelcyn describes the part beginning with this text as "Introduction" (8 folios), and distinguishes it from a "Chapter 1" (he provides no incipits for the chapter beginnings). Can his chapter 1 be deduced from Platonov, or should I merge Introduction and Chapter 1 in the manuscript descriptio?

Reply:
Yes, it can. The text of the edition (St. Petersburg ms) has no indication for the first part, but it has the indication "2nd part" for the following part, so we can deduce that what stands before should be understood as "1st part". Now I see that these are exactly "parts" (sg. ክፍል) and not chapters (ምዕራፍ), so I switched to "part". Ok? I assume "parts" indeed appear in the manuscript, though we cannot check.  Strelcyn speaks of "chapters", but we have no access to the ms, unfortunately, and I am not sure that he would consciously make a distinction between both.